src/server/editorServices.ts

```diff
@@ -609,6 +637,7 @@ module ts.server {
                     }
                 }
                 if (content !== undefined) {
+                    content = this.expandTabs(content);

```

Comment:
Is this changing the data that the host gave us?  If so, won't our lines/characters be off in the responses we send the host?